dbt_project.yml

```diff
@@ -32,8 +33,10 @@ quoting:
 
 # https://docs.getdbt.com/docs/configuring-incremental-models
 models:
+  # TODO: 0.17.0 has an error with persist_docs and interpets it as unused
   persist_docs: #allows for docs to flow through to BigQuery console tables/views
     relation: true
+    columns: true
```

Comment:
perists column desc to bigquery columns

![image](https://user-images.githubusercontent.com/19176976/84206558-04675d80-aa75-11ea-9093-b467b6068dcc.png)